src/containers/BooksForm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const bookCategories = ["Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"];
+
+const BooksForm = () => (
+    <form>
+        <label for="title-input">Title</label>
+        <input id="title-input" type="text"/>
+        <select>
+            {bookCategories.map(category => (
+                <option value="category">{category}</option>
```

Comment:
Lists in react need to have a unique `key` property in order to identify them for performance reasons. See the [docs](https://reactjs.org/docs/lists-and-keys.html#keys). 
So let's add a `key` prop to the mapped component.